makefiles/common.mk

```
@@ -187,7 +187,12 @@ _THEOS_INTERNAL_SEARCHPATHS += \
 	$(TARGET_PRIVATE_FRAMEWORK_PATH) \
 	$(THEOS_VENDOR_LIBRARY_PATH)/$(THEOS_TARGET_NAME)/$(or $(THEOS_PACKAGE_SCHEME),rootful) \
 	$(THEOS_LIBRARY_PATH)/$(THEOS_TARGET_NAME)/$(or $(THEOS_PACKAGE_SCHEME),rootful)
+
+ifeq ($(messages),$(filter $(messages),true 1 yes))
```

Please use the defined variables for each of these

```suggestion
ifeq ($(_THEOS_VERBOSE),$(_THEOS_TRUE))
```